onegini-react-native-sdk.podspec

```diff
@@ -6,18 +6,18 @@ Pod::Spec.new do |s|
   s.name         = package["name"]
   s.version      = package["version"]
   s.summary      = package["description"]
-  s.homepage     = "https://www.npmjs.com/package/react-native-sdk-beta"
+  s.homepage     = "https://www.npmjs.com/package/onegini-react-native-sdk"
 
   s.license      = package['license']
   s.authors      = { "Develocraft" => "valeriy.rumak@develocraft.com" }
```

Comment:
I would replace it with OneWelcome

Reply:
This file will get removed in the split, so we can ignore this.  I will make sure to update it to the correct one though as even the other file has the wrong info there.